### PR TITLE
@W-9643768 updates to flick method to bound the flick co-ordinates

### PR DESCRIPTION
**Pull request by rajukamal**
@lizaiv77 @helenren please review.

### CODE REVIEW

File: utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java [hunk 1]
```diff
@@ -76,11 +78,13 @@ private static int getBoundedCoordinate(int c, int dimension) {
    */
   private static Point boundCoordinates(AppiumDriver driver, Point location) {
     int width, height;
+    Dimension dimension;
     if (isNative(driver)) {
-      width = driver.manage().window().getSize().getWidth();
-      height = driver.manage().window().getSize().getHeight();
+      dimension = driver.manage().window().getSize();
```
Comment (rajukamal) [inline utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java:83]: reduces server calls

File: utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java [hunk 2]
```diff
@@ -172,8 +176,9 @@ static boolean isNative(AppiumDriver driver) {
   private static Point getAbsoluteCoordinates(WebElement webView, int xWebView,
       int yWebView) {
     // The dimensions are all relative to the WebView, so calculate absolute coordinates
-    int webViewX = webView.getLocation().getX();
-    int webViewY = webView.getLocation().getY();
+    Point location = webView.getLocation();
```
Comment (rajukamal) [inline utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java:179]: reduces server calls

File: utam-core/src/main/java/utam/core/selenium/appium/MobileElementAdapter.java [hunk 3]
```diff
@@ -34,9 +35,10 @@ public MobileElementAdapter(WebElement element, Driver driverAdapter) {
   @Override
   public void flick(int xOffset, int yOffset) {
     AppiumDriver appiumDriver = MobileDriverAdapter.getAppiumDriver(driverAdapter);
-    Point nativeStartPoint = getWebElement().getLocation().moveBy(
-        getWebElement().getSize().getWidth() / 2,
-        getWebElement().getSize().getHeight() / 2
+    Rectangle rect = getWebElement().getRect();
```
Comment (rajukamal) [inline utam-core/src/main/java/utam/core/selenium/appium/MobileElementAdapter.java:38]: reduces server calls

**Comment by rajukamal** [comment]
> @lizaiv77 @helenren please review.

Removed flickItems() method from Touchable interface

File: utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java (hunk 1 above)
Comment (helenren) [reply utam-core/src/main/java/utam/core/selenium/appium/MobileDriverUtils.java:83]: 👍

**Review by helenren — APPROVED** [review]

**Review by lizaiv77 — APPROVED** [review]

**Pull request merged**